demo/meeting/src/containers/DevicePermissionControl/DevicePermissionControl.tsx

```diff
@@ -1,34 +1,54 @@
 // Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import { ControlBarButton, Cog, useMeetingManager, Camera, Sound, Dots, DeviceLabels, } from 'amazon-chime-sdk-component-library-react';
+import {
+  ControlBarButton,
+  Cog,
+  useMeetingManager,
+  Camera,
+  Sound,
+  Dots,
+  DeviceLabels,
+} from 'amazon-chime-sdk-component-library-react';
 import React from 'react';
 import DevicePermissionPrompt from '../DevicePermissionPrompt';
 
 const DevicePermissionControl = (props: { deviceLabels: DeviceLabels }) => {
   const meetingManager = useMeetingManager();
+  const { deviceLabels } = props;
 
-  const handleClick = async () => {
-    await meetingManager.invokeDeviceProvider(props.deviceLabels);
-  }
+  const handleClick = async (): Promise<void> => {
+    meetingManager.invokeDeviceProvider(deviceLabels);
```

Comment:
why we remove `await` here?

Reply:
It's a typo. `invokeDeviceProvider` is not an async function, I checked the code.

Reply:
If it does not return a Promise, then you could remove the `async` and change the `Promise<void>` to `void`

Reply:
oh yes, that's right, thank you. Updated.